docs/content/config.md

```diff
@@ -24,14 +23,50 @@ workspaces are created by calling the appropriate function on the IWorkspaceCont
 context.WorkspaceContainer.CreateWorkspace("workspace");
 context.WorkspaceContainer.CreateWorkspace("another");
 context.WorkspaceContainer.CreateWorkspaces("more", "than", "one");
+```
+ 
+
+##  LayoutEngines
+
+Layout engines define the way windows get arranged on each workspace. There are a number of different ways to layout windows and you can swap between them dynamically. By default workspacer loads 2 engines to switch between, the FullLayoutEngine and the TallLayoutEngine, howeveryou can change the set of default layout engines via:
+
+```csharp
+context.DefaultLayouts = () => new ILayoutEngine[] { new FullLayoutEngine() };
+```
+
+ You can even manually specify the LayoutEngine(s) on a per-workspace basis, by passing the different LayoutEngines to the `CreateWorkspace` (or equivalent) function
 
-# you can even define custom layouts for a specific workspace
+```csharp
 context.WorkspaceContainer.CreateWorkspace("layouts!", new FullLayoutEngine(), new TallLayoutEngine());
 ```
 
-## how do I make workspacer ignore some windows? <br/> how do I make some windows always go to a specific workspace?
 
-workspacer will ignore certain system windows, such as Task Manager and workspacer itself, and will by route workspaces to the focused workspace when they are opened. as with most everything else, this is completely customizable via IWindowRouter.
+### Existing LayoutEngines
+* [FullLayoutEngine](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * Maximizes the current focused window and hides all others
+* [TallLayoutEngine](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * Splits the screen in two horizontal zones, 1 primary and one secondary
+    * Windows get created in the secondary zone by default
+    * The number of windows in the primary zone can be dynamically adjusted
+ * [PaneLayoutEngines](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * The PaneLayoutEngine is an abstract class with two implementations: the `VertLayoutEngine` and the `HorzLayoutEngine`
+    * The `VertLayoutEngine` aligns windows in columns, the `HorzLayoutEngine` in rows
+
+
+### In development
+* GridLayoutEngine
+    * Creates a dynamic (NxN) grid in which windows are sorted
+    * Requires support for on-the-fly change in row and column numbers
+* DwindleLayout
+    * Uses primary and secondary zones as the TallLayoutEngine
+    * Tiles windows in a `left-right-bottom-left-up-right-bottom-left-up-` order
```

Comment:
It's merged now! 🎉

Reply:
Sorry I don't think I was clear - I mean that it's still under the `### In development` section